lib/user_app_jwt/Kconfig

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig USER_APP_JWT
+	bool "User Application JWT Library"
+	select BASE64
+	select CJSON_LIB
+	# needed to print integer values in json
+	select CBPRINTF_FP_SUPPORT
+
+if USER_APP_JWT
+
+config JWT_VERIFY_SIGNATURE
+	bool "Verify signature after signing"
+	default y
+
+config PRINT_EXPORTED_PUBKEY_DER
+	bool "Print to terminal the DER formatted public key"
+	default y
+
+config USER_APP_JWT_MAX_LEN
+	int "Buffer size to allocate for JWT"
+	default 900
+	help
+	  Minimum JWT response is ~470 bytes.
+
+config USER_APP_JWT_LOG_LEVEL
```

Comment:
This should be removed and replaced with the Zephyr log config template. See lines 21 and 23 here: https://github.com/nrfconnect/sdk-nrf/blob/b0461ac723c3e66f35c829a62223f9cbc2ea34d0/lib/at_monitor/Kconfig#L21